Version/VersionInfo.cs

```diff
@@ -2,8 +2,9 @@
 
 internal static class VersionInfo
 {
-	public const string Mainversion = "2.0.0"; // Use numbers only or the new version notification won't work
-	public static readonly string RELEASEDATE = "June 25, 2017";
+    // keep this updated at every major release

```

Comment:
It's being updated, just not merged to master. I guess all we need is merging it back after release. @adelikat?

Reply:
It makes no difference what the value here is from a release perspective. This appears to be generated automatically in a release build process anyway (if you check the version of 2.3.0 in bizbox it actually returns the correct version).

Therefore I believe we should just set it in master here so that development versions have access to the current stable version number.

Reply:
It's set manually in the release branch.

Reply:
Ahh ok. Then it doesn't matter how it gets updated in master, as long as it does :)